app/frontend/src/api/models.ts

```diff
@@ -25,13 +25,31 @@ export type AskRequest = {
     idToken?: string;
 };
 
-export type AskResponse = {
-    answer: string;
+export type ResponseMessage = {
```

Comment:
Is this protocol documented somewhere? Wondering if we should include a link to the protocol as a comment for context

Reply:
It's documented in a PR: https://github.com/Azure/azureml_run_specification/pull/1066/files
We could link to that or wait for that to be merged.

Reply:
Up to you - I'm OK with a follow up to add this documentation